server/src/routes/api/v1/reviewsPageRouter.js

```diff
@@ -0,0 +1,17 @@
+import express from "express"
+import { Activity } from "../../../models/index.js"
+
+const reviewsRouter = new express.Router({ mergeParams: true })
+
+reviewsRouter.get("/", async (req, res) => {
```

Comment:
not needed - this should be queried from the show activity endpoint - you will need this router when you add POST functionality